src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -429,19 +429,32 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator, IRBlobSe
                             case "!Plot":
                                 await _callbacks.Plot(
                                     new PlotMessage(
-                                        message.GetString(0, "xaml_file_path"),
-                                        message.GetInt32(1, "active_plot_index"),
-                                        message.GetInt32(2, "plot_count"),
+                                        message.GetString(0, "device_id").SafeParseGuid(),
```

Comment:
I'm changing them all back to `Guid.Parse`, removing the extension method, and handling `null` in the only place where it can be returned, which is where we call `rtvs:::graphics.ide.getactivedeviceid()`.